Content.Client/_NF/M_Emp/UI/M_EmpWindow.xaml

```diff
@@ -0,0 +1,48 @@
+<DefaultWindow xmlns="https://spacestation14.io"
+            Title="{Loc 'm_emp-console-menu-title'}"
+            MinSize="250 250">
+    <BoxContainer Orientation="Vertical">
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'm_emp-menu-note0'}" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text=" " />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Button Name="RequestButton"
+                    Text="{Loc 'm_emp-request-button'}"
+                    StyleClasses="OpenBoth"
+                    Disabled="False" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text=" " />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text=" " />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'm_emp-menu-note1'}" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'm_emp-menu-note2'}" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'm_emp-menu-note3'}" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'm_emp-menu-note4'}" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
+            <Label Text="{Loc 'm_emp-menu-note5'}" />
+        </BoxContainer>
+        <BoxContainer Orientation="Horizontal">
```

Comment:
so alternatively here you would have one box container of pre-defined size and have your text wordwrap here, even with optional scroll bar spacing in case a translator gets extra wordy

Reply:
Having issues fixing this, with my overall xaml knowledge. if you have any idea how to deal with it I will need some help,
Either way I posted about it to the discord also.